tests/integration/components/user-skills-input-test.js

```diff
@@ -20,12 +19,8 @@ let mockStore = {
 };
 
 let mockUserSkillsService = {
-  findUserSkill() {
-    return K;
-  },
-  removeSkill() {
-    return K;
-  }
+  findUserSkill() {},
+  removeSkill() {}
```

Comment:
@sbatson5 I'm not quite sure how to exactly reproduce the previous implementation as it was when it returned `K`.